### PR TITLE
Highlight matched hints

### PR DESCRIPTION
**Pull request by sprintr**
Highlights matched hints in CSS

### CODE REVIEW

File: src/extensions/default/CSSCodeHints/styles/brackets-css-hints.css [hunk 1]
```diff
@@ -1,5 +1,5 @@
 /*
- * Copyright (c) 2014 Adobe Systems Incorporated. All rights reserved.
+ * Copyright (c) 2015 Adobe Systems Incorporated. All rights reserved.
```
Comment (redmunds) [inline src/extensions/default/CSSCodeHints/styles/brackets-css-hints.css]: We never update the copyright date on files. This is the date the file was added to code base.

**Comment by sprintr** [comment]
@redmunds Done!

**Comment by redmunds** [comment]
Good catch. It looks like same change needs to be made to **/src/extensions/default/JavaScriptCodeHints/styles/brackets-js-hints.css**.

**Comment by sprintr** [comment]
JavaScript hints uses more than one rules for hints, depending on priority. It will be a nice idea to have @larz0's opinion on the choice of colors.

**Comment by redmunds** [comment]
@sprintr 

> JavaScript hints uses more than one rules for hints, depending on priority.

Good point. I'll just merge this fix for now.

**Pull request merged**